needles/gnome_software_search_encyclopedia.json

```diff
@@ -0,0 +1,31 @@
+{
+  "tags": [
+    "gnome_software_search",
+    "gnome_software_search_encyclopedia"
+  ],
+  "area": [
+    {
+      "xpos": 0,
+      "ypos": 0,
+      "width": 1024,
+      "height": 768,
+      "type": "match"
+    },
+    {
+      "": "Top bar buttons (updates button appears and disappears)",
```

Comment:
It does?

Reply:
Did for me. I assumed it was only shown when updates were known to be available.

[Yes, it is](https://github.com/endlessm/gnome-software/blob/master/src/gs-shell.c#L826)